src/grpc/concordium_p2p_rpc_grpc_pb.d.ts

```diff
@@ -0,0 +1,722 @@
+// package: concordium
```

Comment:
Is it intentional that these are committed?

Reply:
It was intentional while I was learning how the npm packaging works. I should be able to remove them now.